app/views/root_taxons/_list_children.html.erb

```diff
@@ -0,0 +1,16 @@
+<% (children || []).each do |child| %>
+  <li>
+    <%= child[:name] %>
+    <span class='taxon-size-bar'
+          <%# Set the width based on the size of the child, and the
+              maximum size across the whole tree, and scale this to be
+              between 0 and 30 percent. -%>
+          style='width: <%= (child[:size] * 30) / @taxonomy_size.max_size %>%'>
```

Comment:
This could definitely be extracted to a helper method, maybe in a presenter?

Reply:
I don't know what a presenter is in the context of Rails, so I've had a guess and come up with 0e5ad5f , what do you think?